src/models/VisualStyleModel/impl/visualStyle-utils.ts

```diff
@@ -324,9 +326,7 @@ export const computeView = (
             `Mapping is defined, but Mapper for ${vp.name} is not found`,
           )
         }
-        const computedValue: VisualPropertyValueType = mapper(
-          attributeValueAssigned,
-        )
+        const computedValue: VisualPropertyValueType = mapper(attributeValue)
```

Comment:
Also the attributeValue could be an array of values.  Has validation already been performed or should we do a check here to make sure it is not an array?

Reply:
ValueType includes lists, but yes, we need to handle list types in the implementation in the future versions.